CONTRIBUTORS.md

```
@@ -116,7 +116,7 @@ The following is a list of contributors to the .NET **nanoFramework** project (a
     <td><a href="https://github.com/ChristophLindemann">Christoph Lindemann</a></td>
   </tr>
   <tr>
-    <td><img src="https://github.com/MBNSoftware.png?size=50" height="50" width="50"></td>
+    <td><img src="https://github.com/MBNSoftware.png?size=50" he ight="50" width="50"></td>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)